### PR TITLE
Added support for HitBTC Exchange

### PR DESCRIPTION
**Pull request by claus235**

### CODE REVIEW

File: gradlew.bat [hunk 1]
```diff
@@ -1,84 +1,84 @@
-@if "%DEBUG%" == "" @echo off
```
Comment (rpanak-generalbytes) [inline gradlew.bat]: Try to keep the commit simple. There should be no need to change the `gradlew.bat` file. Please revert the change.

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/BitcoinExtension.java [hunk 2]
```diff
@@ -59,7 +60,8 @@ public IExchange createExchange(String paramString) //(Bitstamp is in built-in e
                 String apiKey = paramTokenizer.nextToken();
                 String apiSecret = paramTokenizer.nextToken();
                 return new BitfinexExchange(apiKey, apiSecret);
-            } if ("bittrex".equalsIgnoreCase(prefix)) {
+            }
+            else if ("bittrex".equalsIgnoreCase(prefix)) {
```
Comment (rpanak-generalbytes) [inline server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/BitcoinExtension.java]: Great you spotted the missing `else`, but please keep the code style consistent. `else if` should start on the same line the previous `if`-block ends on.

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/BitcoinExtension.java [hunk 3]
```diff
@@ -74,6 +76,12 @@ public IExchange createExchange(String paramString) //(Bitstamp is in built-in e
                 }
                 return new ItBitExchange(userId, accountId, clientKey, clientSecret, preferredFiatCurrency);
             }
+            else if("hitbtc".equalsIgnoreCase(prefix)) {
```
Comment (rpanak-generalbytes) [inline server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/BitcoinExtension.java]: Same as previous comment - change to `} else if("hitbtc".equalsIgnoreCase(prefix)) {`

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/hitbtc/HitbtcExchange.java [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+
+    private String accountId;
+
+    public HitbtcExchange(String preferredFiatCurrency) {
+        super(getDefaultSpecification(), preferredFiatCurrency);
+    }
+
+    public HitbtcExchange(String clientKey, String clientSecret, String preferredFiatCurrency) {
+        super(getSpecification(clientKey, clientSecret), preferredFiatCurrency);
+        this.accountId = accountId;
```
Comment (rpanak-generalbytes) [inline server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/hitbtc/HitbtcExchange.java]: unnecessary?

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/hitbtc/HitbtcExchange.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+import org.knowm.xchange.ExchangeSpecification;
+import org.knowm.xchange.dto.account.AccountInfo;
+import org.knowm.xchange.dto.account.Wallet;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class HitbtcExchange extends XChangeExchange {
+
+    private String accountId;
```
Comment (rpanak-generalbytes) [inline server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/exchanges/hitbtc/HitbtcExchange.java]: unnecessary?

File: gradlew.bat (hunk 1 above)
Comment (claus235) [reply gradlew.bat]: Done!
